# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xf86-video-sis/dri.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

From: Julien Cristau <jcristau@debian.org>
Date: Thu, 9 Aug 2007 01:35:12 +0000 (+0200)
Subject: Fix xserver version detection.
X-Git-Url: http://gitweb.freedesktop.org/?p=xorg/driver/xf86-video-sis.git;a=commitdiff;h=391dad44fa305be4cded31cf2f9a4fba7420af99

Fix xserver version detection.

Newer xserver releases report a version of 1.x, which is smaller
than 6.8.99.900.
---

--- a/src/sis.h
+++ b/src/sis.h
@@ -92,7 +92,7 @@
 	(((major) * 10000000) + ((minor) * 100000) + ((patch) * 1000) + snap)
 #define XF86_VERSION_CURRENT XF86_VERSION_NUMERIC(4,3,99,902,0)
 #endif
-#if XORG_VERSION_CURRENT >= XORG_VERSION_NUMERIC(6,8,99,900,0)
+#if XORG_VERSION_CURRENT >= XORG_VERSION_NUMERIC(6,8,99,900,0) || XORG_VERSION_CURRENT < XORG_VERSION_NUMERIC(4,0,0,0,0)
 #define SISISXORG6899900
 #endif
 #if 0
